File: back/back/Models/Renterer.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace back.Models
{
    public class Renterer
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
        public string PhoneNumber { get; set; }
        public string CompanyName { get; set; }
        public string Email { get; set; }
        public string ProfilePictureUrl { get; set; }
    }
}

File: front/src/app/models/Renterer.ts
export class Renterer{
    id:string;
    name: string;
    username: string;
    email: string;
    password: string;
    phoneNumber: string;
    address: string;
    companyName: string;
    profilePictureUrl: string;

    constructor(name, username, email, pass, phone, addr, company, picUrl) {
       this.name=name;
       this.username=username;
       this.email=email;
       this.password=pass;
       this.phoneNumber=phone;
       this.address=addr;
       this.companyName=company;
       this.profilePictureUrl=picUrl;
    }
  
}